// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Throw when timeout is exceeded 1`] = `
"FAIL __tests__/index.test.ts
  Timeout Options
    ✕ Throw when timeout is reached

  ● Timeout Options › Throw when timeout is reached

    Step timed out after 250ms

      65 |
      66 |   const timeoutPromise = new Promise((_, reject) => {
    > 67 |     timer = setTimeout(reject, ms, new Error(\`Step timed out after \${ms}ms\`));
         |                                    ^
      68 |   });
      69 |
      70 |   await Promise.race([promise, timeoutPromise]);

      at ../../src/gherkin/feature.ts:67:36
      at ../../src/gherkin/feature.ts:66:26
      at ../../src/gherkin/feature.ts:8:71
      at Object.<anonymous>.__awaiter (../../src/gherkin/feature.ts:4:12)
      at wrapTimeout (../../src/gherkin/feature.ts:63:66)
      at ../../src/gherkin/feature.ts:50:25
      at ../../src/gherkin/feature.ts:8:71
      at Object.<anonymous>.__awaiter (../../src/gherkin/feature.ts:4:12)
      at Object.<anonymous> (../../src/gherkin/feature.ts:39:31)"
`;
